modules/mobile/spec/request/check_in_request_spec.rb

```diff
@@ -10,7 +10,7 @@
   describe 'POST /mobile/v0/appointments/check-in' do
     it 'correctly updates check in when 200' do
       VCR.use_cassette('chip/authenticated_check_in/post_check_in_200') do
-        VCR.use_cassette('check_in/chip/token/token_200') do
```

Comment:
I could have left this alone. For some reason there are identical cassettes in both chip and checking. Given that we're using chip elsewhere, I chose to be consistent about it.